solr/packaging/test/bats_helper.bash

```diff
@@ -59,7 +59,7 @@ save_home_on_failure() {
 }
 
 shutdown_all() {
-  solr stop -all >/dev/null 2>&1
+  solr stop --all >/dev/null 2>&1
```

Comment:
```suggestion
  solr stop -all >/dev/null 2>&1
```

I think the bats tests should stay as they on this branch, with main focus on regression testing existing opts. Bonus if we add tests that exercise new long-opts as well, but imo not that critical.